plugins/GSdx/GSSetting.cpp

```diff
@@ -46,6 +46,7 @@ const char* dialog_message(int ID, bool* updateText) {
 #endif
 		case IDC_CRC_LEVEL:
 			return "Control the number of Auto-CRC hacks applied to games.\n\n"
+				"Automatic:\nAutomatically select 'Partial' or 'Full' level based on the selected renderer.\n\n"
```

Comment:
> And replace `Accurate/depth options may be required with Blending Unit Accuracy/Depth Emulation may be need to be enabled` at OpenGL.

I'd need to keep the recommended info in the tooltip if I'm to make this change. Otherwise I think it wouldn't make as much sense.